### DISCUSSION TITLE
using slideWidth and setUpSlides()

### DISCUSSION THREAD

**Discussion by timbotimber** [glidejs/glide#594] [Q&A]
Hi there, loving the slider! I have a question regarding the size component: https://glidejs.com/docs/components/sizes/

How can I use the slideWidth value and setUpSlides() method to dynamically set the slide width before the slider loads? I'm currently having to set it with CSS and it takes an extremely long time to get it working correctly when one of the slides has an active property. 

Let me know if I can provide any other information to help with this.

**Comment by BuddyLReno** [comment]
There _really_ needs to be an example for this. I'm stuck with this too.
